### DISCUSSION TITLE
Ongoing Priorities

### DISCUSSION THREAD

**Discussion by dshoneisda** [finos/common-domain-model#3106] [General]
Survey link here: https://www.surveymonkey.com/r/JKXGCBN

At the SWG in July it was noted that it would be useful to understand the community's priorities for a range of contributions and enhancements to the model.
Some of these items have already been prioritised by the TAWG for funding. Some items such as Serialisation Phase 1 are not mentioned as they have already been prioritised and have been or are in the process of going ahead.

Please select as many items as you see as necessary priorities for increased adoption, stability and growth of the CDM.
If you have ideas/items on your list that are cross-model in nature (e.g. are not product specific- these items are done via proposals in the relevant WGs) and do not appear in the poll, please add to comments on the github issue that brought you to this poll.

This poll does not cover extensions such as ISDA's DRR- these should be handled through existing forums.

N.B. Presence of an item on this list does not guarantee funding. This poll is being used as a barometer for interest

**Comment by dshoneisda** [comment]
Survey is now closed

Results:

![image](https://github.com/user-attachments/assets/9e4d79f2-46b6-4c33-8cad-40aa24d5cfca)
![image](https://github.com/user-attachments/assets/95e62034-67e5-4799-8323-2031b0690092)
![image](https://github.com/user-attachments/assets/9f3efdee-83ea-4b2b-a823-daf1088b88ad)

[cdmsurveyaug24.xlsx](https://github.com/user-attachments/files/17010301/cdmsurveyaug24.xlsx)


To be discussed at next SWG

**Comment by lolabeis** [comment]
Results presented at #3117
